documentation/dev/api/swagger.json

```diff
@@ -0,0 +1,123 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "prompb/rpc.proto",
+    "version": "version not set"
+  },
+  "schemes": [
+    "http",
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/admin/v1/tsdb/delete_series": {
```

Comment:
I had presumed that a v2 api would include everything, in particular the current querying stuff, so users don't have to worry about which version to use for which features.